pkgs/development/python-modules/pytest-alembic/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  alembic,
+  pytest,
+  sqlalchemy,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-alembic";
+  version = "0.11.1";
+
+  src = fetchFromGitHub {
+    owner = "schireson";
+    repo = "pytest-alembic";
+    rev = "v${version}";
+    hash = "sha256-A+Nl+DAK8uhC3iOgS1i6x/nj9BJ0My3lpft+pRdoab8=";
+  };
+  pyproject = true;
+  nativeBuildInputs = [ poetry-core ];
```

```suggestion
  build-system = [ poetry-core ];
```
use build-system for build-system.requires/setup_requires